analyzers/dart-analyze/.deepsource/issues/DRT-W1710.toml

```diff
@@ -0,0 +1,41 @@
+
+title = "Setters must declare exactly one required positional parameter"
```

Comment:
There are some of these issue TOMLs where the 1st line is empty. Not sure if this can cause some issue in the parsing. Let's remove this anyway?

Reply:
Shouldn't cause a problem since armory parses the issues successfully, but anyways removed the leading newline from all issue toml and also update `issue_gen.py` to not in include a leading newline.